src/libdrakvuf/vmi.c

```diff
@@ -464,12 +464,8 @@ event_response_t int3_cb(vmi_instance_t vmi, vmi_event_t *event) {
     else
         event->interrupt_event.reinject = 0;
 
-    proc_data_t proc_data = {
-        .name   = NULL,
-        .pid    = 0,
-        .ppid   = 0,
-        .userid = 0,
-    };
+    proc_data_t proc_data = {};
```

Comment:
While I think `{}` null-initializes the struct `{0}` would make that more obvious.

Reply:
Are you sure `{0}` initialize all of the members to null or just the first one letting the rest with whatever values?

Reply:
Reference C99 Standard 6.7.8.21:

> If there are fewer initializers in a brace-enclosed list than there are elements or members of an aggregate, or fewer characters in a string literal used to initialize an array of known size than there are elements in the array, the remainder of the aggregate shall be initialized implicitly the same as objects that have static storage duration.

Reply:
Also, empty braces seem to be only a GNU extension so some compilers may not honor it.

Reply:
Ok, give me a few minutes...